RecoTracker/TransientTrackingRecHit/python/TTRHBuilderWithTemplate_cfi.py

```diff
@@ -8,6 +8,8 @@
                                                                                Matcher = 'StandardMatcher',
                                                                                ComputeCoarseLocalPositionFromDisk = False)
 
+TTRHBuilderAngleAndTemplateWithoutProbQ = TTRHBuilderAngleAndTemplate.clone(ComponentName = 'WithAngleAndTemplateWithoutProbQ')
+
 from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
 trackingPhase2PU140.toModify(TTRHBuilderAngleAndTemplate, 
```

Comment:
based on the failures in tests being all in phase-2 wfs
I guess a `trackingPhase2PU140.toModify(TTRHBuilderAngleAndTemplateWithoutProbQ, ` is missing here ... and the `PixelCPEGeneric.toModify` (despite of it kind of being obviously without ProbQ).

Reply:
> and the PixelCPEGeneric.toModify (despite of it kind of being obviously without ProbQ)

The point of that modifier is to replace template reco with generic, when the former is not available (e.g. certain phase2 pixel sensor technologies) , so if `TTRHBuilderAngleAndTemplateWithoutProbQ` becomes the CPE of choice also for phase2, the modifier should replace that one too (as you correctly did in https://github.com/cms-sw/cmssw/pull/39776/commits/2bde976035ea7024bafbcd2c030ba53495e1ef42)